### PR TITLE
Use latest akka snapshot

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/build.sbt [hunk 1]
```diff
@@ -20,7 +20,7 @@ Global / cancelable := false // ctrl-c
 // For akka management snapshot
 resolvers += Resolver.bintrayRepo("akka", "snapshots")
 
-val AkkaVersion = "2.6.9"
+val AkkaVersion = "2.6.9+70-58b1c30c"
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/build.sbt:23]: where is the akka.io snapshots resolver?

Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/build.sbt:23]: nope, just noticed it

Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/build.sbt:23]: also need it in the other projects

Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/build.sbt:23]: hmm, it has 

```scala
resolvers += Resolver.bintrayRepo("akka", "snapshots")
```

Comment (patriknw) [reply docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/build.sbt:23]: Akka snapshots are at another place, to confuse people ;-) https://doc.akka.io/docs/akka/current/project/links.html#snapshots-repository

**Comment by octonato** [comment]
I'm merging this since is green and will unlock other PRs.

**Pull request merged**